### PR TITLE
Update gpdb pgvector docs for new 0.5 featurtes

### PR DESCRIPTION
**Pull request by dyozie**

### CODE REVIEW

File: gpdb-doc/markdown/ref_guide/modules/pgvector/pgvector.html.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Refer to [Installing Additional Supplied Modules](../../../install_guide/install
 
 ## <a id="using"></a>About the vector Types, Operators, and Functions
 
-`pgvector` provides a `vector` data type and an index access method named `ivfflat`. This type and method and the supporting functions and operators provided by the module enable you to to perform exact and approximate neighbor search on, and determine L2, inner product, and cosine distance between, embeddings. You can also use the module to store and query embeddings.
+`pgvector` provides a `vector` data type and an new index access methods `ivfflat` and `hnsw`. The type, methods, and the supporting functions and operators provided by the module enable you to perform exact and approximate neighbor search on, and determine L2, inner product, and cosine distance between, embeddings. You can also use the module to store and query embeddings.
```
Comment (lisakowen) [inline gpdb-doc/markdown/ref_guide/modules/pgvector/pgvector.html.md:21]: an new -> new ?

**Review by lisakowen — APPROVED** [review]
i think we are missing the element-wise multiplication operator.

**Review by fanfuxiaoran — APPROVED** [review]

File: gpdb-doc/markdown/ref_guide/modules/pgvector/pgvector.html.md (hunk 1 above)
Comment (radarwave) [reply gpdb-doc/markdown/ref_guide/modules/pgvector/pgvector.html.md:21]: I think we can remove 'an new' here.

**Review by radarwave — APPROVED** [review]
Minor comment, others looks good to me.

**Pull request merged**